api/v1alpha3/types.go

```diff
@@ -516,12 +516,35 @@ type Instance struct {
 	// Indicates whether the instance is optimized for Amazon EBS I/O.
 	EBSOptimized *bool `json:"ebsOptimized,omitempty"`
 
-	// Specifies size (in Gi) of the root storage device
-	RootDeviceSize int64 `json:"rootDeviceSize,omitempty"`
+	// Configuration options for the root storage volume
+	RootVolumeOptions *RootVolumeOptions `json:"rootVolumeOptions,omitempty"`
```

Comment:
```suggestion
	RootVolume *RootVolumeOptions `json:"rootVolume,omitempty"`
```

Reply:
This also need `//+optional`

Reply:
You want this renaming all over? s/RootVolumeOptions/RootVolume/g

Reply:
Yeah, `Options` is redundant given that this field is already within `spec:`